c++/src/kj/test.h

```diff
@@ -57,6 +57,8 @@ class TestCase {
     }
   }
 
+  static size_t iterCount();
```

Comment:
Why did you move this from private to protected? It doesn't appear that anything else in this PR uses this?